airlock/business_logic.py

```diff
@@ -535,6 +557,9 @@ def output_files_set(self):
             for request_file in filegroup.output_files
         }
 
+    def supporting_files_count(self):
+        return len(self.all_files_set()) - len(self.output_files_set())
```

Comment:
will this count `RequestFileType.WITHDRAWN` files as supporting files?

Reply:
oh, yes, it will, will fix thanks